packages/flutter_test/lib/src/binding.dart

```diff
@@ -959,6 +968,10 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   late bool _beforeTestCheckIntrinsicSizes;
 
   void _verifyInvariants() {
+    // subclasses overrides this
```

Comment:
nit: This comment threw me off when I read it. I read this as "subclasses are intended to override this method" not as "existing subclasses in this library already override this method", which had me confused as to why we were intending users to override a private method. More explicit verbiage as to why we're keeping this method would be appreciated.

Reply:
+1

Reply:
Done!